### ISSUE TITLE
Montecarlo dropout

### ISSUE THREAD

**Issue by felixbur** [felixbur/nkululeko#137]
nkululeko already knows the concept of "runs", i.e. running the same experiment n times with different random initializations.
Would be  interesting to output the ensemble performance of all these models.

**Comment by bagustris** [comment]
If I understand correctly, it is similar to an ensemble of different seeds.
Last time, I edit svm and balancing to use the same seed so the output will be consistent for every run.

If so, the ini file is the same for different results, hence we can use the output prediction CSV file to make ensemble?

**Comment by felixbur** [comment]
yes, it's very similar, the variation comes from the random drop outs.
I'd like to look into it, if it could be just an additional flag in the ini file

**Comment by felixbur** [comment]
i think this is solved by the ensemble funtionality
